phoenicis-tools/src/main/java/org/phoenicis/tools/processes/PhoenicisProcessBuilder.java

```diff
@@ -0,0 +1,99 @@
+package org.phoenicis.tools.processes;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A process builder to be used from inside the scripts
+ */
+public class PhoenicisProcessBuilder {
```

Comment:
In the DTO's, the DTO and the builder are in the same file. Should it be the same here?

Reply:
I think the code is better readable if we split both classes in two separate files. But I'm fine either way.

Reply:
I don't care too much but it might make sense to be consistent.

Reply:
After thinking some more about this I'm not sure this would work, because I don't know if it is possible to access inner Java classes from JavaScript.